solutions/devsprint-bruno-faganello-2/SpaceApp/Utils/SPConstants.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  SPConstants.swift
+//  SpaceApp
+//
+//  Created by Paolo Prodossimo Lopes on 25/04/22.
+//
+
+import UIKit
+
+struct SPConstants {
+    
+    struct SPTabBar {
```

Comment:
[NICE TO HAVE]
Remove Unnecessary code

Reply:
I deleted this file, for now is useless.